styles/themes/classic_grey.css

```diff
@@ -750,6 +750,26 @@ table.snapshottable th {
 .bad-char {
   background: #ff99cc;
 }
+/* ------------------------------------------------------------------------ */
+/* Star texts */
+.star-gold {
+  font-size: 3em;
+  color: #FFD700;
+  text-align: center;
+  text-shadow: 3px 2px 5px black;
+}
+.star-silver {
+  font-size: 3em;
+  color: gray;
```

Comment:
I think the lighter 'silver' colour looks better.

Reply:
I think so too and have changed it.